### PR TITLE
merge harvey-go into harvey

### PR DESCRIPTION
**Pull request by rminnich**
we are going to make Go a first class citizen, might as well bring in harvey-go.

### CODE REVIEW

**Review by dancrossnyc — APPROVED** [review]

**Comment by gmacd** [comment]
I'd argue we don't necessarily want all the go programs in tree.  Some yes, but only those that are generally useful.  I'd also be wary of the ipfs cmd - it's really cool, but IIRC it takes ages to build.

File: sys/postpipe.go [hunk 1]
```diff
@@ -0,0 +1,34 @@
+// Copyright 2020 The Ninep Authors.  All rights reserved.
```
Comment (floren) [inline sys/postpipe.go:1]: I don't think we want this in /sys/postpipe.go

File: README.md [hunk 2]
```diff
@@ -0,0 +1 @@
+# Go Packages
```
Comment (floren) [inline README.md:1]: we should get rid of this README file, it'll only conflict with the real one

File: sys/src/cmd/preen/.gitignore [hunk 3]
```diff
@@ -0,0 +1 @@
+/preen
```
Comment (floren) [inline sys/src/cmd/preen/.gitignore:1]: this isn't right

Comment (rminnich) [reply sys/src/cmd/preen/.gitignore:1]: why?
it's saying the preen binary in this directory should be ignored.

File: sys/postpipe.go (hunk 1 above)
Comment (rminnich) [reply sys/postpipe.go]: moved do sys/src/cmd/postpipe.go

File: README.md (hunk 2 above)
Comment (rminnich) [reply README.md]: removed

File: sys/src/cmd/preen/.gitignore (hunk 3 above)
Comment (rminnich) [reply sys/src/cmd/preen/.gitignore:1]: If the pattern contains a slash, which is not in the trailing position (it's not the last character), then it is matched only with pathnames relative to the .gitignore file location.

**Review by floren — APPROVED** [review]

**Pull request merged**